panel/pane/vtk/vtk.py

```diff
@@ -134,6 +134,29 @@ def get_color_mappers(self):
                     LinearColorMapper(low=low, high=high, name=name, palette=palette)
                 )
         return cmaps
+    
+    def construct_colorbars(self, orientation='horizontal'):
```

Comment:
Did you want to deprecate this? I have no opinion but if you think it's a good idea we should do it now and add a warning using `self.param.warning()` which explains it's deprecated and what the replacement is.

Reply:
No need to deprecate it for the moment.